pages/book/cells.mdx

```diff
@@ -113,11 +112,42 @@ contract SerializationExample {
 }
 ```
 
-### `remaining` [#serialization-bytes64]
+### `remaining` [#serialization-remaining]
+
+To illustrate what `remaining{:tact}` serialization type does, let's take a look at [cell manipulation primitives](#cells-immutability) and their [TL-B][tlb] representation produced by Tact:
+
+```tact
+contract SerializationExample {
```

Comment:
code snippets should be mostly compilable,
this contract lacks variable initializers or `init`